packages/storage-documents/src/on-demand-page-scan-result.ts

```diff
@@ -5,7 +5,7 @@ import { ItemType } from './item-type';
 
 export declare type ReportFormat = 'sarif';
 export declare type ScanState = 'unknown' | 'pass' | 'fail';
-export declare type OnDemandPageScanRunState = 'unknown' | 'accepted' | 'queued' | 'running' | 'completed' | 'failed';
+export declare type OnDemandPageScanRunState = 'accepted' | 'queued' | 'running' | 'completed' | 'failed';
```

Comment:
you also need to fix the get404ErrorResponse method in azure function. that is returning unknown value for run.